src/Symfony/Component/Serializer/Mapping/Loader/schema/dic/serializer-mapping/serializer-mapping-1.0.xsd

```diff
@@ -47,10 +47,11 @@
         Contains serialization groups for a attributes. The name of the attribute should be given in the "name" option.
       ]]></xsd:documentation>
         </xsd:annotation>
-        <xsd:sequence>
+        <xsd:sequence minOccurs="0">
```

Comment:
IMO it's not necessary: previously metadata was only used for groups. It makes sense to require it.
But with this new annotation (only in 3.1+) you can use metadata for groups or max depth. It makes sense to make groups optional only for 3.1+.